### PR TITLE
"Stable Arvo" Blog post

### PR DESCRIPTION
**Pull request by urcades**
Would appreciate a quick look-over to make sure my metadata is OK and the blog-index description is good. I think it might be a bit too long, could use some editing down.

### CODE REVIEW

File: content/blog/stable-arvo.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
++++
+title = "Stable Arvo"
+date = 2019-11-14
+description = "In order to get closer to an Urbit that can last forever, we set for ourselves the goal of getting Arvo to a point that we can credibly call ‘stable’ by the end of 2019. "
```
Comment (matildepark) [inline content/blog/stable-arvo.md]:
```suggestion
description = "This year we set out to get Arvo to a point that we can credibly call ‘stable.' "
```

File: content/blog/stable-arvo.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+aliases = ["/posts/essays/stable-arvo", "/posts/stable-arvo"]
```
Comment (matildepark) [inline content/blog/stable-arvo.md:5]: Aliases are only necessary if the post was at a previous place!

File: content/blog/stable-arvo.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+With the recent 0.9.0 release we’ve gotten to the point where we can make almost all our upgrades over the air. Even the language is now upgradeable over the wire.
+
+But — in order for that scene in 2059 to actually work, these upgrade mechanisms have to be in place. As your ship upgrades itself, the system needs a way to ensure that each upgrade gets applied in a compatible way. Right now we’re careful about deploying over the air updates such that they don’t break, but that’s obviously not sustainable.
+
+Once this versioning scheme gets released later this year, there’s nothing technically preventing you from starting a late-2019 Urbit in 2039. We hope you’ll use it a lot in the interim — but this is an essential feature for a decentralized system that claims to upgrade itself. We’re really excited to get it implemented.
+
+These three areas of work are all due to complete later this quarter. This gets us something that’s much more resilient than what we’ve ever had before. Given a stable system, we can actually start to put some serious weight on it. 
+
+`~poldec-tonteg`
```
Comment (matildepark) [inline content/blog/stable-arvo.md]: Signoff isn't necessary, we use bylines now.

**Comment by matildepark** [comment]
Everything else looks dope.

File: content/blog/stable-arvo.md (hunk 2 above)
Comment (urcades) [reply content/blog/stable-arvo.md:5]: AH, gotcha — very good to know

File: content/blog/stable-arvo.md (hunk 3 above)
Comment (urcades) [reply content/blog/stable-arvo.md]: Perfect, another good detail to note for future updates.

**Comment by jfranklin9000** [comment]
"relies on event" -> "relies on an event"

lines 56 and 57 should be combined into one long line?

**Pull request merged**